README.md

```diff
@@ -39,6 +39,12 @@ In this project a hambuger button is built so that that the user clicks on the m
 - Twitter: [@NguforEmmanuella](https://twitter.com/NguforEmmanuella)
 - LinkedIn: [Nguforemmanuella](https://linkedin.com/in/Nguforemmanuella)
 
+👤 **Nejjari001**
+
+- GitHub: [@Ngufor-emmanuella](https://github.com/Ngufor-emmanuella)
+- Twitter: [@NguforEmmanuella](https://twitter.com/NguforEmmanuella)
+- LinkedIn: [Nguforemmanuella](https://linkedin.com/in/Nguforemmanuella)
+
```

Comment:
- [ ] [OPTIONAL] I noticed that these details are repeated. Please use your coding partner's correct details 🙏